### ISSUE TITLE
False positives for `no-constant-condition`

### ISSUE THREAD

**Issue by blakeembrey** [buzinas/tslint-eslint-rules#86]
Errors on increment/decrement.

```ts
let len = 10

while (len--) {}
```

**Comment by buzinas** [comment]
The error was fixed and it will not happen anymore in the next release.

**Comment by buzinas** [comment]
@blakeembrey v1.6.0 released
